Dockerfile

```diff
@@ -1,12 +1,13 @@
 ARG GDAL=ubuntu-small-3.3.3
 FROM osgeo/gdal:${GDAL} AS gdal 
 ENV LANG="C.UTF-8" LC_ALL="C.UTF-8"
+ARG PYTHON_VERSION=3.8
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-    python3 python3-pip python3-dev python3-venv cython3 g++ && \
+    python${PYTHON_VERSION} python3-pip python${PYTHON_VERSION}-dev python${PYTHON_VERSION}-venv cython3 g++ && \
```

Comment:
I just tried with python 3.7 and it doesn't work. It seems that `python3.7-dev` doesn't exist

Reply:
I only tested with 3.8 and 3.9 and both of those worked. 3.8 is the default python3 on Ubuntu 20, so that would make sense.

Might not be bad to drop Python 3.7 since numpy will be dropping it in December: https://github.com/pyproj4/pyproj/issues/930

Reply:
deadsnakes ppa added